lib/minfraud.rb

```diff
@@ -29,16 +29,24 @@
 
 module Minfraud
   class << self
+    # @!attribute account_id
+    # @return [String] The MaxMind account ID that is used for authorization
+    attr_accessor :account_id
+
     # @!attribute host
     # @return [String, nil] The host to use when connecting to the web service.
     attr_accessor :host
 
     # @!attribute user_id
-    # @return [String] MaxMind account ID that is used for authorization
+    # @return [String] The MaxMind account ID that is used for authorization.
+    #
+    # Deprecated. This exists for backwards compatibility. Please use
+    # account_id instead. When we do a major version release, we should remove
```

Comment:
Does this show up in the generated documentation? If so, maybe we should phrase it as, "This will be removed in the next major version."

Reply:
Yeah, good idea. Thanks!